### PR TITLE
feat(core): add fallbackvalues to flags

### PR DESCRIPTION
**Pull request by cstrnt**

### CODE REVIEW

**Comment by cstrnt** [comment]
Current dependencies on/for this PR:
* main
  * **PR #91** <a href="https://app.graphite.dev/github/pr/tryabby/abby/91" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/tryabby/abby/91?utm_source=stack-comment).

File: packages/core/src/index.ts [hunk 1]
```diff
@@ -301,9 +304,20 @@ export class Abby<
 
     const defaultValue = this._cfg.settings?.flags?.defaultValue;
 
-    // return the defaultValue if exists
-    if (storedValue === undefined && defaultValue != null) {
-      return defaultValue;
+    if (storedValue === undefined) {
```
Comment (slaesh) [inline packages/core/src/index.ts:307]: cleaning stuff may help here..

```suggestion
    if (storedValue !== undefined) { do the normal stuff }
    
    // fallback?
    
    // default?
```

**Review by slaesh — APPROVED** [review]

File: packages/core/tests/base.test.ts [hunk 2]
```diff
@@ -115,7 +115,14 @@ describe("Abby", () => {
         config1: "String",
         config2: "String",
       },
+      flags: ["flag1", "flag2"],
       settings: {
+        flags: {
+          fallbackValues: {
```
Comment (slaesh) [inline packages/core/tests/base.test.ts:121]: for which ENV is this? all? we would need to have it for each ENV independently :)

**Comment by slaesh** [comment]
I am not sure why FF's and RC's are handled differently.. but they are, somehow.. so, how does this change can be applied to the RC part? it cant.. we should revert this asap before its too late! ;D

**Pull request merged**